lib/beaker/subcommand.rb

```diff
@@ -61,5 +63,51 @@ def provision()
         say "Hosts have already been provisioned"
       end
     end
+
+    desc 'exec', 'execute a directory, file, or beaker suite'
+    long_desc <<-LONG_DESC
+    Run a single file, directory, or beaker suite. If supplied a file or directory,
+    that resource will be run in the context of the `tests` suite; If supplied a beaker
+    suite, then just that suite will run. If no resource is supplied, then this command
+    executes the suites as they are defined in the configuration.
+    LONG_DESC
+    def exec(resource=nil)
+      if options[:help]
+        invoke :help, [], ["exec"]
```

Comment:
We could pass `--help` as a `class_option` flag. That will allow all subcommands to process it and simply invoke `beaker help *subcommand*` and then return without actually executing that subcommands. I'm not particularly fond of how the help ends up printing the options; it ends up adding a  rather ugly usage that doesn't seem easy to reformat.

```
Options:
  [--help], [--no-help]

Beaker run options:
  -h, [--hosts=HOSTS]
...
```
I really hate that `[--no-help]` malarky, but, I don't see any real way of configuring the help print function to not do [that](https://github.com/erikhuda/thor/blob/master/lib/thor/parser/option.rb#L91). So...I guess I'm fine with help just print like that for now, unless we want to hide it, so that `--help` is an easter egg?